index.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+const ToDo = require('./todo/model.js');
+const PORT = process.env.PORT || 2000;
+const express = require('express');
+const app = express();
+const server = require('./lib/server');
+const jsonParser = require('body-parser').json();
+const routes = require('./todo/route');
+
+
+app.post('/api/todos', jsonParser, routes.post);
+
+app.get('/api/todos', routes.get);
+
+app.delete('/api/todos', routes.delete);
```

Comment:
I would use the express.Router construct instead, more industry standard and would clean up the code a little bit.